.github/workflows/main.yml

```diff
@@ -100,13 +92,4 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/setup
       - name: Lint Swift using SwiftLint
-        run: bundle exec rake lint:swift:swiftlint
-
-  swiftformat:
-    name: "Lint Swift (SwiftFormat)"
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/setup
-      - name: Lint Swift using SwiftFormat
-        run: bundle exec rake lint:swift:swiftformat
+        run: bundle exec rake lint:swift
```

Comment:
I combined the two linters into a single job again, since now the test job is that one that takes the longest to run once started